packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -571,30 +584,33 @@ mod default {
     pub fn builtin_debug_hook_fallback<'a>(
         frame: &Frame,
         ctx: &mut HookContext<'a, Frame>,
-    ) -> Option<Emit> {
+    ) -> Vec<Emit> {
+        #[allow(unused_mut)]
+        let mut emit = vec![];
+
         // we're only able to use `request_ref` in nightly, because the Provider API hasn't been
         // stabilized yet.
         #[cfg(nightly)]
         {
             #[cfg(feature = "std")]
             if let Some(bt) = frame.request_ref() {
-                return Some(backtrace(bt, ctx.cast()));
+                emit.append(&mut backtrace(bt, ctx.cast()));
```

Comment:
With the above changes, this will become
```suggestion
                emit.push(backtrace(bt, ctx.cast()));
```